plugins/catalog-backend/src/service/CatalogBuilder.ts

```diff
@@ -722,6 +722,24 @@ export class CatalogBuilder {
 
     this.checkMissingExternalProcessors(processors);
 
+    // Lastly sort the processors by priority. Config can override the
+    // priority of a processor to allow control of 3rd party processors.
+    processors.sort((a, b) => {
+      const aPriority =
+        config.getOptionalNumber(
```

Comment:
Alright - should we rebase this on top of latest master now that https://github.com/backstage/backstage/pull/31034 is merged, and break out this prio reading into a helper function that does try/catch to be safe in the same way?